apps/state/lib/state/schedule.ex

```diff
@@ -259,6 +260,13 @@ defmodule State.Schedule do
     do_time_filter(schedules, 0, max)
   end
 
+  defp do_post_search_filter(schedules, %{remove_unsupported_times: true}) do
```

Comment:
I don't think that you'll actually do this filter if there's a min or max time.

Reply:
Moved it to the view as per Paul's comment.